emperor/support_files/js/emperor.js

```diff
@@ -886,20 +888,23 @@ function toggleTaxaLabels(){
 	}
 }
 
-/* Turn on and off the spheres representing the biplots on screen */
-function toggleBiplotVisibility(){
+/* Turn on and off the spheres representing the biplots on screen
+
+If the toggleArrow variable is true, then only the visibility of the taxa vectors will be altered
+Otherwise, only the visibility of the */
```

Comment:
This line should still be removed. And the line above should be aligned with the word `Turn` that's at the beginning of this block of comments.